lottie-swift/src/Public/Animation/AnimationView.swift

```diff
@@ -797,6 +797,16 @@ final public class AnimationView: LottieView {
     self.animationContext = nil
   }
   
+  fileprivate func removeAllAnimations() {
+    guard animationContext != nil else { return }
+    let pauseFrame = realtimeAnimationFrame
+    for animationName in animationLayer?.animationKeys() ?? [] {
```

Comment:
With `forEach`one doesn't have to deal with a default values for optionals. This makes the code less prone to a wrong default values.
**Solution:** 
`animationLayer?.animationKeys()?.forEach { animationLayer?.removeAnimation(forKey: $0) }`